client/functions.lua

```diff
@@ -12,8 +12,27 @@ function QBCore.Functions.GetCoords(entity)
     return vector4(coords.x, coords.y, coords.z, GetEntityHeading(entity))
 end
 
+--- QBCore.Functions.HasItem checks if a player has the specified `items` in their inventory
+--- with the specified `amount`.
+---
+--- @param items string|string[]    The item(s) to check for. Can be a string or a table.
+--- @param amount integer           The desired quantity of each item.
+---
+--- @return boolean Returns true if the player has the specified items in the desired quantity,
+---                 false otherwise
 function QBCore.Functions.HasItem(items, amount)
-    return exports['qb-inventory']:HasItem(items, amount)
+    local count = exports.ox_inventory:Search('count', items)
+
+    if type(items) == 'table' and type(count) == 'table' then
+        for _, v in pairs(count) do
+            if v < amount then
```

Comment:
should amount be optional? So that if the caller doesn't specify an amount it defaults to any?